### PR TITLE
[SHELL32] Refresh 'General' tab if folder icon updated

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
In the folder property sheet, if the folder icon was changed, then 'General' tab should be also updated.
JIRA issue: [CORE-11407](https://jira.reactos.org/browse/CORE-11407)

## Proposed changes

- Implement PSM_QUERYSIBLINGS action for each property page.
- Send PSM_QUERYSIBLINGS message to the property sheet when the icon was updated.

### CODE REVIEW

File: dll/win32/shell32/dialogs/filedefext.cpp [hunk 1]
```diff
@@ -717,6 +715,19 @@ CFileDefExt::GeneralPageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lPar
             }
             break;
         }
+        case PSM_QUERYSIBLINGS:
+        {
+            // reset icon
+            HWND hIconCtrl = GetDlgItem(hwndDlg, 14025);
```
Comment (HBelusca) [inline dll/win32/shell32/dialogs/filedefext.cpp]: Do we have a resource constant for this hardcoded number?

Comment (katahiromz) [reply dll/win32/shell32/dialogs/filedefext.cpp]: No, yet. I will add them.

Comment (binarymaster) [reply dll/win32/shell32/dialogs/filedefext.cpp]: @HBelusca looks like file properties dialog is totally missing any resource constants: https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/shell32/lang/en-US.rc;hb=fafcd50a35a77f1dd1f4606a30dd4f5b6fe520bd#l261

Comment (katahiromz) [reply dll/win32/shell32/dialogs/filedefext.cpp]: As a second option, you can use the macro constants in <dlgs.h>.

Comment (HBelusca) [reply dll/win32/shell32/dialogs/filedefext.cpp]: OK let's do the following: I commit this PR now, and let's create (if you wish) a separate PR where the constants are introduced?

Comment (katahiromz) [reply dll/win32/shell32/dialogs/filedefext.cpp]: Okay, let's go with it.

**Pull request merged**